docs/features/source-generators.cookbook.md

```diff
@@ -620,6 +620,110 @@ public class MyGenerator : ISourceGenerator
 }
 ```
 
+### Unit Test generators
```

Comment:
maybe a dumb nit: consider titling this 'Unit testing of generators' so nobody gets the notion at a glance that this is about writing a source generator which generates unit test cases